src/Microsoft.AspNet.OData.Shared/Builder/PropertyKind.cs

```diff
@@ -36,6 +36,12 @@ public enum PropertyKind
         /// <summary>
         /// Represents a dynamic property dictionary for an open type.
         /// </summary>
-        Dynamic = 5
+        Dynamic = 5,
+
+        /// <summary>
+        /// Represents a instance annotation for a CLR type.
+        /// </summary>
+        InstanceAnnotations = 6
```

Comment:
>InstanceAnnotations [](start = 8, length = 19)

Maybe just name it as "Annotation"

Reply:
Wanted to call out InstanceAnnotations , @mikepizzo - whats your suggestion

Reply:
I like "Instance" as it differentiates from metadata annotations, and the plural, since the property holds multiple instance annotations. 

So I would keep it as it is.


---
In reply to: [489038130](https://github.com/OData/WebApi/pull/2219#discussion_r489038130) [](ancestors = 489038130)